src/main/java/com/intellij/compiler/notNullVerification/ThrowOnNullMethodVisitor.java

```diff
@@ -64,6 +65,14 @@ private void setInstrumented() {
         instrumented = true;
     }
 
+    /** This will be invoked only when visiting bytecode produced by java 8+ compiler with '-parameters' option. */
+    @Override
+    public void visitParameter(String name, int access) {
+        if (parameterNames == null) parameterNames = new ArrayList<>(argumentTypes.length);
```

Comment:
i've seen statements like this in the repo, so figured it's ok (and more readable for single trivial statements - imho, of course).